doc/datasets/rcv1.rst

```diff
@@ -32,7 +32,7 @@ The target values are stored in a scipy CSR sparse matrix, with 804414 samples a
 Each sample can be identified by its ID, ranging (with gaps) from 2286 to 810596::
 
     >>> rcv1.sample_id[:3]
-    array([2286, 2287, 2288], dtype=int32)
```

Comment:
There is a fixture to skip the test - `doc/datasets/rcv1_fixture.py` - to avoid download.